_data/sites.json

```
@@ -7222,6 +7222,19 @@
         ]
     },
 
+    {
+        "name": "FREAKHOSTING",
+        "url": "https://freakhosting.com/clientarea/clientarea.php",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
Since it requires somewhat reaching customer support, we'll put it as hard